ui/js/controllers/perf/graphs.js

```diff
@@ -642,6 +648,7 @@ perf.controller('GraphsCtrl', [
                     series.relatedAlertSummaries = [];
                     var repo = $rootScope.repos.find(repo =>
                         repo.name === series.projectName);
+
```

Comment:
Can you remove this newline before landing?

Reply:
Sure.